deploy-board/deploy_board/static/js/components/clusterconfigcomponents.js

```diff
@@ -189,6 +189,11 @@ Vue.component('aws-user-data', {
                 acc[entry.name] = entry.value;
                 return acc;
             }, {});
+
+            // hide link for cmp_base cluster profiles
+            if (userDataObj['pinfo_cluster'] && userDataObj['pinfo_role'] === 'cmp_base') {
```

Comment:
just ` userDataObj['pinfo_role'] === 'cmp_base'` to hide is enough? why we need pinfo_cluster?

Reply:
updated, thanks. I originally thought this was just for the cmp_base images in the cluster hiera subdirectory.